src/virtualship/cli/commands.py

```diff
@@ -15,7 +17,26 @@
 )
 def init(path):
     """Entrypoint for the tool."""
-    raise NotImplementedError("Not implemented yet.")
+    path = Path(path)
+    path.mkdir(exist_ok=True)
+
+    config = path / SHIP_CONFIG
+    schedule = path / SCHEDULE
+
+    if config.exists():
+        raise FileExistsError(
+            f"File '{config}' already exist. Please remove it or choose another directory."
```

Comment:
"directory" or "path"?

Reply:
Directory. The `path` varname is the path to the directory